### PR TITLE
[Arm backend] Fix for TOSA BI clamp ops

### PR DESCRIPTION
**Pull request by freddan80**
Min/max range values need to be on quantized form.

### CODE REVIEW

File: backends/xnnpack/test/tester/tester.py [hunk 1]
```diff
@@ -555,10 +555,7 @@ def run_method_and_compare_outputs(
             print(f"Run {run_iteration} with input shapes: {input_shapes}")
 
             # Reference output (and quantization scale)
-            (
-                reference_output,
-                quantization_scale,
-            ) = self._calculate_reference_output(
+            (reference_output, quantization_scale,) = self._calculate_reference_output(
```
[the remark this answers is not in the document]
Comment (freddan80) [reply backends/xnnpack/test/tester/tester.py:558]: hmm, the CI lintrunner seem to think differently than my local one

[the remark this answers is not in the document]
Comment (freddan80) [reply backends/xnnpack/test/tester/tester.py:558]: Weird. I use the same version, 0.11.0, as mandated by the contribution guidelines. I had to re-run `lintrunner init` to get the same result. I'll push the update soon...

**Comment by freddan80** [comment]
CI failures not related to this PR.

File: backends/arm/operators/op_hardtanh.py [hunk 2]
```diff
@@ -30,12 +32,31 @@ def define_node(
         is_quant_node: bool,
     ) -> None:
         attr = ts.TosaSerializerAttribute()
+
+        if is_quant_node:
+            # Get quant parameters
+            scale, zp = get_quant_node_args(node.all_input_nodes[0])
+            # Convert to quantized representation
+            clamp_min_qs = round((inputs[1].number / scale) + zp)
+            clamp_min_qs = max(clamp_min_qs, -128)
```
Comment (digantdesai) [inline backends/arm/operators/op_hardtanh.py:41]: we should get qmin/qmax from the quant node

**Review by digantdesai — APPROVED** [review]

**Pull request closed without merging**